app-k9mail/src/main/kotlin/app/k9mail/provider/K9AppNameProvider.kt

```diff
@@ -4,15 +4,20 @@ import android.content.Context
 import app.k9mail.core.common.provider.AppNameProvider
 import app.k9mail.core.common.provider.BrandNameProvider
 import com.fsck.k9.R
+import com.fsck.k9.preferences.FilePrefixProvider
 
 class K9AppNameProvider(
     context: Context,
-) : AppNameProvider, BrandNameProvider {
+) : AppNameProvider, BrandNameProvider, FilePrefixProvider {
     override val appName: String by lazy {
         context.getString(R.string.app_name)
     }
 
     override val brandName: String by lazy {
         context.getString(R.string.app_name)
     }
+
+    override val filePrefix: String by lazy {
+        "k9"
+    }
```

Comment:
There's nothing that would benefit from computing this value lazily. So this can simply be:

```suggestion
    override val filePrefix: String = "k9"
```